packages/client-core/src/admin/components/Project/ProjectFields.tsx

```diff
@@ -209,7 +209,7 @@ const ProjectFields = ({ inputProject, existingProject = false, changeDestinatio
         }
       } else {
         ProjectUpdateService.setShowCommitSelector(project, false)
-        ProjectUpdateService.setBranchError(project, projectResponse.text)
+        ProjectUpdateService.setBranchError(project, (projectResponse as any).text)
```

Comment:
Why using as any here? I think in after feathers migration we have type object of response.